lib/gollum-lib/markups.rb

```diff
@@ -1,20 +1,58 @@
 # ~*~ encoding: utf-8 ~*~
 module Gollum
+  module MarkupRegisterUtils
+    # Check if a gem exists. This implementation requires Gem::Specificaton to
+    # be filled.
+    def gem_exists? name
```

Comment:
Style-wise, I prefer parentheses around method arguments (`gem_exists?(name)`) just to keep it consistent with the rest of the gollum code base.